cmd/tke-installer/app/installer/installer.go

```diff
@@ -2434,6 +2434,9 @@ func (t *TKE) writeKubeconfig(ctx context.Context) error {
 }
 
 func (t *TKE) patchPlatformVersion(ctx context.Context) error {
+	if t.globalClient == nil {
+		return errors.New("can't get cluster client")
+	}
```

Comment:
please don't hard code tke version(1.5) in code,  this will cause patch version overwrite main release version,   we should define default  tke version(1.5) in code, but use env `VERSION`  to overwrite, thanks.